net-analyzer/nuttcp/files/nuttcp-8.1.4-makefile.patch

```diff
@@ -0,0 +1,35 @@
+--- a/Makefile	2017-10-29 08:39:34.209474792 +0200
++++ b/Makefile	2017-10-29 08:49:24.964638395 +0200
+@@ -1,9 +1,8 @@
+ APP = nuttcp-8.1.4
+ #EXTRAVERSION=-pre1
+-CC = gcc -Wall
++CC += gcc -Wall
```

Comment:
Did you mean "?=" instead of "+="?

Reply:
Yes, but it is not needed because I override CC in ebuild.